datacube_stats/statistics.py

```diff
@@ -314,6 +314,37 @@ def measurements(self, input_measurements):
                 index_measurements + text_measurements)
 
 
+class Percentile(PerBandIndexStat):
+    def __init__(self, q):
+        if isinstance(q, list):
```

Comment:
Probably doesn't matter, but I'd use `isinstance(q, collections.Sequence)`, just in case.